src/v2/spec.yaml

```diff
@@ -1804,6 +1924,93 @@ paths:
                 $ref: "#/components/schemas/FeedResponse"
         "400":
           $ref: "#/components/responses/400Response"
+  /farcaster/frame:
```

Comment:
Should we add the frame list API?

Reply:
so external developers can get a list of frames they created? It will also return the frames in frame studio at this point because our db does not separate between programmatically created frames and the ones made in the studio